NEWPACKAGE.md

```diff
@@ -0,0 +1,20 @@
+# Request to Include a New Package in Fedora CoreOS
+
+If you would like to propose the inclusion of a new package into the base
+content set of Fedora CoreOS, please open a [new issue](https://github.com/coreos/fedora-coreos-tracker/issues/new) 
+here on the [fedora-coreos-tracker repo](https://github.com/coreos/fedora-coreos-tracker) 
```

Comment:
Hmm, given that we link to the new issue, probably OK to just drop this entire line.

Reply:
Yeah, not sure why I thought to be overly verbose